### PR TITLE
Contributing: fix typos

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //-------------------------------------------------------------
-// <copyright company=�Microsoft Corporation�>
-//   Copyright � Microsoft Corporation. All Rights Reserved.
+// <copyright company=�Microsoft Corporation�>
+//   Copyright � Microsoft Corporation. All Rights Reserved.
```
Comment (lambdageek) [inline mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs]: This doesn't look right.  Please convert the file from Windows 1252 to UTF-8

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 //-------------------------------------------------------------
-// <copyright company=�Microsoft Corporation�>
-//   Copyright � Microsoft Corporation. All Rights Reserved.
+// <copyright company=�Microsoft Corporation�>
+//   Copyright � Microsoft Corporation. All Rights Reserved.
```
Comment (lambdageek) [inline mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs]: Hm. Looks like these files are in the Windows-1252 encoding and your changes are in some other encoding.  Could you convert them to utf-8 instead?

File: mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs [hunk 3]
```diff
@@ -1017,7 +1017,7 @@ private static bool ProcessSimplifyCase_EliminateWhenClauses(RuleProcessingConte
         ///     WHEN W1 THEN T1 
         ///     WHEN W2 THEN T2 ... 
         ///     ELSE (CASE 
-        ///             WHEN WN1 THEN TN1, � 
+        ///             WHEN WN1 THEN TN1, �
```
Comment (lambdageek) [inline mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs]: Looks like the file was in Windows-1252 originally (that should be a '…' there). Please convert it to utf-8.

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs (hunk 1 above)
Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs]: My mistake. I will fix them. Thank you for the review.

Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs]: That's weird. It's already UTF-8.

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs (hunk 2 above)
Comment (jaykrell) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs]: How about just sticking to 7 bit ASCII?

Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs]: I can't because I'm using either notepad ++ or VS Code.

Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs]: image

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs (hunk 1 above)
Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs]: image

File: mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs (hunk 3 above)
Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs]: image

Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs]: @lambdageek I made sure it is UTF-8. I delete that weird character in order to fix it.

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs (hunk 1 above)
Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs]: @lambdageek I made sure it is UTF-8. I replaced those weird characters with their previous values in order to fix it.

File: mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs (hunk 2 above)
Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs]: @lambdageek I made sure it is UTF-8. I replaced those weird characters with their previous values in order to fix it.

File: mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs [hunk 4]
```diff
@@ -1096,7 +1096,7 @@ static bool ProcessComparisonsOverConstant(RuleProcessingContext context, Node n
 
             // Make sure that the terminal character of the pattern is a '%' character. Also
             // ensure that this character does not occur anywhere else. And finally, ensure
-            // that the pattern is atmost one character longer than the string itself
+            // that the pattern is almost one character longer than the string itself
```
Comment (akoeplinger) [inline mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs]: this should actually be `at most`, not almost.

Comment (MaherJendoubi) [reply mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs]: My bad. I fix it. Thanks for the review.

**Pull request merged**